tests/end2end/playwright/form-filter.spec.js

```diff
@@ -30,10 +30,8 @@ test.describe('Form filter', () => {
         await expect(page.locator(countFeature)).toHaveText('1');
 
         let getMapRequest = await getMapPromise;
-        // Re-send the request with additionnal echo param to retrieve the WMS Request
-        let echoGetMap = await page.request.get(getMapRequest.url() + '&__echo__');
-        let originalUrl = decodeURIComponent(await echoGetMap.text());
-        let urlObj = new URLSearchParams((new URL(originalUrl).search));
+        // Re-send the request with additionnal echo param to retrieve the WMS Request params
```

Comment:
```suggestion
        // Re-send the request with additional echo param to retrieve the WMS Request params
```

Re-send the request with additional "echo" parameter to retrieve the query string ? and all other occurences ?